bin/cli.js

```diff
@@ -236,6 +247,36 @@ function invoke(env) {
         .catch(exit);
     });
 
+  commander
+    .command('migrate:print')
+    .description('Print all migrations in directory')
+    .option(
+      '--direction',
+      'Specify migration direction to print. It can be "up" or "down"'
+    )
+<<<<<<< HEAD
```

Comment:
should this be committed?..

Reply:
sorry @kibertoad this was a wip commit. I will notify you when we finish the changes requested.